lighthouse-cli/test/fixtures/perf/delayed-element.js

```diff
@@ -28,11 +28,16 @@ async function rerender(iterations) {
 setTimeout(() => {
   const imgEl = document.createElement('img');
   imgEl.src = '../dobetterweb/lighthouse-480x318.jpg';
-  const textEl = document.createElement('span');
+  const textEl = document.createElement('div');
   textEl.textContent = 'Sorry!';
+  textEl.style.height = '18px' // this height can be flaky so we set it manually
   const top = document.getElementById('late-content');
-  top.appendChild(imgEl);
-  top.appendChild(textEl);
+
+  // Use shadow DOM to verify devtoolsNodePath resolves through it
```

Comment:
these couple lines tweak an existing smoketest to have some shadow DOM so that we don't need to be testing chromestatus. (I also noticed that its pretty flaky)